.idea/runConfigurations.xml

```diff
@@ -3,10 +3,11 @@
   <component name="RunConfigurationProducerService">
     <option name="ignoredProducers">
       <set>
+        <option value="com.android.tools.idea.compose.preview.runconfiguration.ComposePreviewRunConfigurationProducer" />
```

Comment:
There's a change here in  .idea

Reply:
I missed this among all other files. Reverted now

Reply:
Verified and Resolved